src/shared/GitHub.Tests/GitHubHostProviderTests.cs

```diff
@@ -121,7 +121,7 @@ public async Task GitHubHostProvider_GetSupportedAuthenticationModes(string uriS
         [Theory]
         [InlineData("https://example.com", null, "0.1", false, AuthenticationModes.Pat)]
         [InlineData("https://example.com", null, "0.1", true, AuthenticationModes.Basic | AuthenticationModes.Pat)]
-        [InlineData("https://example.com", null, "100.0", false, AuthenticationModes.Browser | AuthenticationModes.Pat)]
+        [InlineData("https://example.com", null, "100.0", false, AuthenticationModes.OAuth | AuthenticationModes.Pat)]
```

Comment:
This seems like an errant revert of a previous change.

Reply:
Oh, I see. `OAuth` is the combination of two. Should we have lines for `Browser`, `Device`, _and_ `OAuth` on this test?